include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -339,7 +342,7 @@ ORT_API_STATUS(OrtGetStringTensorDataLength, _In_ const OrtValue* value, _Out_ s
  * \param value A tensor created from OrtCreateTensor... function.
  * \param s_len total data length, get it from OrtGetStringTensorDataLength
  */
-ORT_API_STATUS(OrtGetStringTensorContent, _In_ const OrtValue* value, _Out_ void* s, size_t s_len,
+ORT_API_STATUS(OrtGetStringTensorContent, _In_ const OrtValue* value, _In_ void* s, size_t s_len,
                _Out_ size_t* offsets, size_t offsets_len);
 
```

Comment:
The documentation for this method is not helpful enough. 
Is 's' _In_Out_ as it's not allocated by OrtGetStringTensorContent? 
Should it explicitly say that s_len is the length allocated in 's'?

Reply:
It's an error when I reverted/merged the code. I'll drop this change. Thanks for helping me find it.